README.md

```diff
@@ -24,3 +24,30 @@ This extension contributes the following settings:
 * `colorTabs.config`:  list of mappings from path to color         
 "default": [{"regex": ".*\/web\/.*", "color": "#ff0000"},{"regex": ".*\/mobile\/.*", "color":  "#00ff00"}],
 
+* Tab borders are colored by default, you can turn this off by adding `"colorTabs.tabsBorder" : false` to settings.json 
+
+* You can also enable title background coloring. This can be done by adding `"colorTabs.titleBackground" : true` to settings.json
+
+For example: 
+
+```
+// To settings.json  => workspace settings. Defaults to the last color (.* is a wildcard).
```

Comment:
Not sure I would put this as the example. If you're trying to say that the last regex wins, we can write that without an example IMO.

However writing a small example is a good idea